Source/Hurl.BrowserSelector/Windows/MainWindow.xaml.cs

```diff
@@ -127,33 +127,33 @@ private void LinkCopyBtnClick(object sender, RoutedEventArgs e)
     }
 
     private void SettingsBtnClick(object sender, RoutedEventArgs e) => Process.Start("explorer", "\"" + Constants.APP_SETTINGS_MAIN + "\"");
-    private void Draggable(object sender, MouseButtonEventArgs e) => this.DragMove();
+    private void Draggable(object sender, MouseButtonEventArgs e) => DragMove();
     private void CloseBtnClick(object sender, RoutedEventArgs e) => MinimizeWindow();
 
     private void MinimizeWindow()
     {
-        this.WindowState = WindowState.Minimized;
-        this.Hide();
+        WindowState = WindowState.Minimized;
+        Hide();
     }
 
-    public void MaximizeWindow()
+    public void ShowWindow()
     {
-        this.Show();
         PositionWindowUnderTheMouse();
```

Comment:
Keep `Show` call on the top..... else, the window will only show up in background